vulnerabilities.yaml

```diff
@@ -1978,3 +1978,34 @@
     special effects in a shell command line), the ``mailcap.findmatch()``
     function now refuses to inject ASCII characters other than alphanumerics
     and ``@+=:,./-_`` into the returned command line.
+
+- name: "Windows: vulnerable zlib 1.2.11"
+  slug: update-zlib-1-2-11
+  cve:
+    - CVE-2018-25032
+  bpo: 47194
+  links:
+    - https://access.redhat.com/security/cve/cve-2018-25032
+  fixed-in:
+    - '3.10': 16a809ffb7af14898ce9ec8165960d96cbcd4ec3
+    - '3.9':  0f0f85e9d8088eb789cda35477900df32adff546
+    - '3.8':  7ccdec3d1d837b910cd4fc5525ecde71a1326202
+    - '3.7':  387f93c156288c170ff0016a75af06e109d48ee1
+  affected-versions:
+    - '3.10'
+    - '3.9'
+    - '3.8'
+    - '3.7'
```

Comment:
I don't think that this list is correct not useful, the tool should compute it automatically.

Reply:
Well, this seems to be a redundant member and we should remove it. Maybe we should change template at the top of vulnerabilities.yaml as well.
```yaml
# Template:
# - name: ""
...
#   fixed-in:
#    - '3.x': commit_sha1
#    - '3.y': commit_sha1
#   affected-versions:
#    - "x.y"
#   description: |
```